clientunaryinterceptor.go

```diff
@@ -62,3 +63,15 @@ func NewClientUnaryInterceptor(opts *Options) grpc.DialOption {
 
 	return grpc.WithUnaryInterceptor(interceptor.Unary())
 }
+
+func NewClientInterceptor(opts *Options) grpc.UnaryClientInterceptor {
```

Comment:
Please, add common / basic comments like for NewClientUnaryInterceptor and NewClientInterceptor quite close to NewClientUnaryInterceptor by implementation; there is a bit redundancy (some kind of copy/paste). Maybe better is to refactor and implement NewClientUnaryInterceptor via NewClientUnaryInterceptor (or via another privat common func for the both), having excluded the redundancy.